### PR TITLE
Add ObservableEither

### PR DESCRIPTION
**Pull request by pheis**

### CODE REVIEW

File: src/ObservableEither.ts [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+ */
+export function leftIO<E, A>(me: IO<E>): ObservableEither<E, A> {
+  return leftObservable(R.fromIO(me))
+}
+
+/**
+ * @since 0.6.8
+ */
+export function fromTaskEither<E, A>(t: TE.TaskEither<E, A>): ObservableEither<E, A> {
+  return defer(t)
```
Comment (mlegenhausen) [inline src/ObservableEither.ts]: You can use `fromTask` from `Observable`

File: src/ObservableEither.ts [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+/**
+ * @since 0.6.8
+ */
+export const swap: <E, A>(ma: ObservableEither<E, A>) => ObservableEither<A, E> = T.swap
+
+/**
+ * @since 0.6.8
+ */
+export const observableEither: Monad2<URI> & Bifunctor2<URI> & MonadObservable2<URI> = {
```
Comment (mlegenhausen) [inline src/ObservableEither.ts]: You can also add `Alt2` and `MonadThrow2`

**Review by mlegenhausen — CHANGES_REQUESTED** [review]
Good starter for a `ObservableEither` can be the base for adding more functionality like `tryCatch`.

File: src/ObservableEither.ts (hunk 1 above)
Comment (pheis) [reply src/ObservableEither.ts]: Yeah!

**Review by mlegenhausen — APPROVED** [review]

**Comment by mlegenhausen** [comment]
@gcanti could you please merge and release? Thanks

**Comment by gcanti** [comment]
@pheis thanks

@mlegenhausen released (thanks for reviewing)

**Pull request merged**